frontend/packages/knative-plugin/src/types.ts

```diff
@@ -197,3 +197,9 @@ export enum DomainMappingResponseAction {
 export enum ServerlessBuildStrategyType {
   ServerlessFunction = 'ServerlessFunction',
 }
+
+export enum ServiceTypeValue {
+  Functions = 'Functions',
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  'Non-functions' = 'Non-functions',
+}
```

Comment:
It's an enum that should not be related to the labels that are displayed to the user. So you should clean this up:

```suggestion
export enum ServiceTypeValue {
  Function = 'Function',
  Service = 'Service',
}
```

If the value is what you render to the user, you should change that in that component.